ras/hwinfo.py

```diff
@@ -49,17 +49,27 @@ def test_list(self):
             cmd = "hwinfo %s" % list_item
             self.run_cmd(cmd)
 
-    def test_disk(self):
+    def test_listmd(self):
+        lists = self.params.get('listmd', default=['--all', '--cpu', '--disk'])
+        for list_item in lists:
+            cmd = "hwinfo %s" % list_item
+            self.run_cmd(cmd)
+
+    def test_only(self):
```

Comment:
we can drop this test cases

Reply:
dropped the listmd testcase as suggested, thanks.